index.html

```diff
@@ -0,0 +1,726 @@
+<!DOCTYPE html>
+<html lang="zxx">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Newsweek &ndash; News, Analysis, Policy, Business, Technology</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <link rel="stylesheet" href="./styles/main.css">
+    </head>
+    <body>
+        <nav class="responsive-nav navbar navbar-expand-lg navbar-dark bg-danger flex-wrap pb-lg-0 px-1">
```

Comment:
Wrap the two `nav` into a `header`
The content of that can be wrap into a `<div class="container">`

Reply:
Why did you use `responsive-nav` and `bg-danger` instead of setting `background-color:red` ?

Reply:
I have made header and footer changes.

As I mentioned above, there is no difference between those 2. Even if danger and red are slightly different shades,  they are only colors which don't make a difference to the layout. On the other hand, since this is a bootstrap project, there should be less emphasis of specifying these details through the CSS. The point should be to write as less CSS as possible, and making use of the Bootstrap classes.